### DISCUSSION TITLE
Hibernate Question

### DISCUSSION THREAD

**Discussion by AbdulJabbar64** [l33t-c0d3r-66/Health-Insurance-Management-System#4] [Q&A] [answered]
I am new to hibernate so can you explain line 12 in https://github.com/l33t-c0d3r-66/Health-Insurance-Management-System/blob/master/src/main/resources/hibernate.cfg.xml

**Comment by l33t-c0d3r-66** [comment, marked as answer]
It will show the SQL query that is being executed by Hibernate in console.

**Reply by AbdulJabbar64** [reply]
Ok Thanks for your response by help of you I got it
